CHANGELOG.md

```diff
@@ -31,6 +32,8 @@
 - xo-acl-resolver v0.4.0
 - xo-server v5.30.0
 - xo-web v5.30.0
+- xo-ova v0.2.0 (renamed from xo-vmdk-to-vhd)
+- xo-upload-ova v0.1.0
```

Comment:
During the release process, packages version are incremented in order, please always add dependencies above.